src/content/docs/developer/modules/other/external-qa-checks.mdx

```diff
@@ -0,0 +1,246 @@
+---
+title: External QA Check Module
+description: Add an external QA check for advanced AI-powered verifications.
+slug: developer/crowdin-apps-module-external-qa-check
+sidebar:
+  label: External QA Check
+---
+
+This module allows for the integration of advanced AI-powered QA checks, enabling verification of translations for nuanced issues that cannot be detected by [default QA checks](/enterprise/project-settings/qa-checks/) or JavaScript-based [Custom QA checks](/enterprise/custom-qa-checks/#creating-and-adding-custom-qa-checks).
```

Comment:
Let's remove the `#creating-and-adding-custom-qa-checks' hash from the link. The user would be interested in the general description of custom QA checks

Reply:
Removed